src/main/java/teammates/storage/sqlentity/Course.java

```diff
@@ -18,24 +18,23 @@
 public class Course extends BaseEntity {
 
     @Id
-    @Column(name = "id", unique = true, nullable = false)
     private String id;
 
-    @Column(name = "name", nullable = false)
```

Comment:
Wanted to ask the reason for deleting `nullable=false`, since the default is `false` which means this column can be null?